### PR TITLE
[LocalTimeSystem] Plugin tests

### LINKED ISSUE #3477: [LocalTimeSystem] Unit tests
Local Time System plugin needs unit tests.

## Testing
in the LocalTimeSystem plugin folder, inside of pluginSpec.js make the main describe an fdescribe and run the tests

### PR DESCRIPTION
**Pull request by jvigliotta**
Added test spec for local time system.

closes #3477 

## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? By its nature
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? Yes

### CODE REVIEW

**Review by davetsay — CHANGES_REQUESTED** [review]
The tests are fine. Is it possible to test at the plugin level though?

```
openmct = createOpenMct();
openmct.install(openmct.plugins.LocalTimeSystem());
```

**Comment by jvigliotta** [comment]
@davetsay added plugin level tests

File: src/plugins/localTimeSystem/pluginSpec.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    beforeEach((done) => {
+
+        openmct = createOpenMct();
+
+        openmct.install(openmct.plugins.LocalTimeSystem());
+
+        openmct.on('start', done);
+        openmct.startHeadless();
+
+        localTimeSystem = openmct.time.timeSystem('local', {
```
Comment (davetsay) [inline src/plugins/localTimeSystem/pluginSpec.js:51]: better to use `LOCAL_SYSTEM_KEY ` as you've defined

File: src/plugins/localTimeSystem/pluginSpec.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+    afterEach(() => {
+        return resetApplicationState(openmct);
+    });
+
+    describe("system", function () {
+
+        it("is installed", () => {
+            let timeSystems = openmct.time.getAllTimeSystems();
+            let local = timeSystems.find(ts => ts.key === 'local');
```
Comment (davetsay) [inline src/plugins/localTimeSystem/pluginSpec.js:69]: better to use `LOCAL_SYSTEM_KEY ` as you've defined

File: src/plugins/localTimeSystem/pluginSpec.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+            expect(local).not.toEqual(-1);
+        });
+
+        it("can be set to be the main time system", () => {
+            expect(openmct.time.timeSystem().key).toBe('local');
```
Comment (davetsay) [inline src/plugins/localTimeSystem/pluginSpec.js:75]: better to use `LOCAL_SYSTEM_KEY ` as you've defined

File: src/plugins/localTimeSystem/pluginSpec.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            expect(localTimeFormatter.validate(JUNK)).toBe(false);
+        });
+    });
+
+    describe("formatter can be obtained from the telemetry API and", () => {
+        let formatter;
+
+        beforeEach(() => {
+            formatter = openmct.telemetry.getFormatter(LOCAL_FORMAT_KEY);
+            console.log(formatter);
```
Comment (davetsay) [inline src/plugins/localTimeSystem/pluginSpec.js:115]: remove `console.log`

**Review by davetsay — CHANGES_REQUESTED** [review]
couple minor fixes

**Review by davetsay — APPROVED** [review]
LGTM

## Reviewer Checklist
| | |
| --- | :---: |
| Changes appear to address issue? | Y |
| Appropriate unit tests included? | Y |
| Code style and in-line documentation are appropriate? | Y |
| Commit messages meet standards? | Y |
| Has associated issue been labelled `unverified`? | Y |

**Pull request merged**